src/internal-server/app/Controllers/Http/ServiceController.js

```diff
@@ -205,19 +192,6 @@ class ServiceController {
     });
   }
 
-  async icon({ params, response }) {
-    const { id } = params;
-
-    const iconPath = path.join(Env.get('USER_PATH'), 'icons', id);
-    if (!fs.existsSync(iconPath)) {
-      return response.status(404).send({
-        status: "Icon doesn't exist",
-      });
-    }
-
-    return response.download(iconPath);
-  }
-
```

Comment:
Is this not needed anymore?
I mean, the previous code will make sure this "shouldn't" happen. But, nevertheless, shouldn't we do this check?

Just asking to make sure 👍

Reply:
that action/method got moved into the `ImageController`